src/probnum/diffeq/_perturbsolve_ivp.py

```diff
@@ -28,12 +28,13 @@ def perturbsolve_ivp(
     y0,
     rng,
     method="RK45",
-    perturb="step-uniform",
+    perturb="step-lognormal",
```

Comment:
this is a better default, because uniform perturbation only works for steps in (0, 1). lognormal perturbation always works.